index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   agent:
+  - apiVersion: v2
+    appVersion: 3.1.0
+    created: "2023-11-14T20:28:59.863924964+06:00"
+    description: Helm chart for the Optimizely Agent https://docs.developers.optimizely.com/full-stack/docs/optimizely-agent
+    digest: 8c95f9dad10753db0d94d56fa25d65c6514e2c07f4df133638f8c799c5260d6f
+    home: https://docs.developers.optimizely.com/full-stack/docs/optimizely-agent
```

Comment:
This docs URL 302s.  

I think this should be https://docs.developers.optimizely.com/full-stack-experimentation/docs/optimizely-agent .